src/components/Button/Button.scss

```diff
@@ -12,10 +12,24 @@
   border: 0.1rem solid $button-border;
   border-radius: $button-border-radius;
 
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
   &:active {
     border: 0.2rem solid $white;
   }
 }
+.button--image--discover-icon{
```

Comment:
You have two classes here, but I don't think it's modifying an existing class?

`.button__discover-icon` ?